### ISSUE TITLE
Adding an import to a component Dart script can cause runtime errors in the template

### ISSUE THREAD

**Issue by srawlins** [dart-lang/angular#529]
This issue comes from https://github.com/dart-lang/markdown/issues/171.

Basically, if you have a component Dart script, like [todo_list_component.dart](https://github.com/google/stagehand/blob/master/templates/web-angular/lib/src/todo_list/todo_list_component.dart), and add a simple import line like:

```dart
import 'package:markdown/markdown.dart';
```

then IntelliJ has nothing to complain about (no .template.dart file open), but you get runtime errors:

```
Observatory listening at http://127.0.0.1:49632/
EXCEPTION: type 'Text' is not a subtype of type 'Node' of 'node' where
Text is from package:markdown/src/ast.dart
Node is from dart:html

STACKTRACE:
#0 Node.append (dart:html:30264)
#1 ViewTodoListComponent0.build (package:untitled2/src/todo_list/todo_list_component.template.dart:123:22)
```

Basically, Markdown's members like Node and Text, are clashing with `dart:html`'s members.

AFAICT, the only way to solve this (at develop-time, without training new users), is to import `dart:html` under a namespace, like `_html`.

**Comment by matanlurey** [comment]
Duplicate of #153.
